raiden/transfer/channel.py

```diff
@@ -1671,11 +1674,18 @@ def handle_receive_lock_expired_light(
             payment_id,
             1,
             state_change.lock_expired,
-            True,
+            False,
```

Comment:
This method is used on target.handle_lock_expired_light, to handle the ReceiveLockExpiredLight state_change.

The ReceiveLockExpiredLight state change is generated on message_handler.handle_message_lockexpired. 

That call is generated on message_handler.on_message, basically when you receive a LockExpired.

According to that flow, i dont understand why you think this isnt related with reception from LC.

Reply:
Because in this case is also used on initiator_manager.handle_lock_expired_light, we added that because it was missing, and because of that we have the partner state being updated with the balance proof, before this that state was always none in the case of mediated payments.

Reply:
This msg is stored on true because is a signed message from the partner, and the lc should respond with a delivered and processed.